scripts/templates/values.yaml

```diff
@@ -36,64 +22,58 @@ statefulset:
       install4jAddVmParams: "-Xms2703m -Xmx2703m"
       user: nexus
       password: nexus
-      jdbcUrl: jdbc:postgresql://localhost:5432/nexus
+      jdbcUrl: jdbc:postgresql://your_database_server:5432/nexus
       nexusInitialPassword: nexusabc
+  livenessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 30
+    path: /
+    periodSeconds: 30
+  readinessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 30
+    path: /
+    periodSeconds: 30
+  imagePullSecret: {}
 ingress:
   name: nexus-ingress
-  enabled: false
-  #host: "example.com" #host to apply this ingress rule to. Uncomment this in your values.yaml and set it as you wish
-  dockerSubdomain: false # uncomment this if you'd like to create a wildcard rule in your ingress for docker sub domain routing. You should also make sure your create routing rules in your load balancer for sub domains of your host. Thus, you must also uncomment and set the host
-  annotations:
-    kubernetes.io/ingress.class: nginx
+  enabled: "false" #intentionally a string because it's used to configure operator via env variable
+  host: "example.com"
+  defaultRule: "false"
+  additionalRules: null
+  tls:
+    enabled: "false" #intentionally a string because it's used to configure operator via env variable
+    secretName: "tlsSecretName"
+  dockerSubdomain: false
+  annotations: null
   dockerIngress:  #Ingress for Docker Connector - comment out if you don't use docker repositories
     name: nexus-docker-ingress
-#    host: "example.com"
-    enabled: false
-    annotations:
-      kubernetes.io/ingress.class: nginx
-workDir:
-  configMapName: create-nexus-workdir-config
-  daemonSetName: create-nexus-work-dir
-  workDirCreator:
-    initContainer:
-      image:
-        repository: ubuntu
-        tag: 23.04
-    container:
-      image:
-        repository: busybox
-        tag: 1.33.1
+    host: "example.com"
+    enabled: "false" #intentionally a string because it's used to configure operator via env variable
+    annotations: null
 nexusData:
   pv:
     directoryPrefix: /var/nexus-repo-mgr-work-dir/work
   pvc:
     size: 100Gi
```

Comment:
100 gigs seems really big, is that truly necessary?

Reply:
Especially since `ReadWriteOnce` is `the volume can be mounted as read-write by a single node.`